src/components/Fetch.js

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import './Fetch.css';
+
+export default function Fetch({ value, onSearchItem }) {
+  const [listItem, setListItem] = useState([]);
+
+  useEffect(() => {
+    loadItems();
+    async function loadItems() {
+      try {
+        const response = await fetch(
+          'https://fetch-me.vercel.app/api/shopping/items'
+        );
+        const data = await response.json();
+        setListItem(data.data);
+      } catch (error) {
+        console.error(error);
+      }
+    }
+  }, []);
```

Comment:
I would probably do the fetching of the data in the `App.js`. It is easier to handle the data and your component would be a bit cleaner.